examples/file-io.rs

```diff
@@ -0,0 +1,58 @@
+//! Run ruperf against some file calls
+//!
+//! This is a small program to run ruperf against. Write and read a file. This 
+//! should call `syscall_write` and `syscall_read`. 
+//! 
+//! The test environment should not need to be a create and should only include
+//! std::* libs.
+//!
+//! Run `make clean` to remove the file.
+//!
+//! Usage:
+//!     ruperf stat ./file-io
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+
+static WRITE_STRING: &str = "The quick brown fox jumped over the lazy dog\n";
+
+fn main() {
+    let writes = 10000;
+    let path = Path::new("read-write-test.txt");
```

Comment:
Can this also create a directory for the tmp file to go into, maybe using std::env::tmp_dir?